go/v1/api/occurrence_test.go

```diff
@@ -904,14 +904,18 @@ func vulnzOcc(t *testing.T, pID, noteName, imageName string) *gpb.Occurrence {
 			Vulnerability: &vpb.Details{
 				PackageIssue: []*vpb.PackageIssue{
 					{
-						AffectedLocation: &vpb.VulnerabilityLocation{
-							CpeUri:  "cpe:/o:debian:debian_linux:8",
-							Package: "abc",
-							Version: &pkgpb.Version{
+						AffectedCpeUri:  "cpe:/o:debian:debian_linux:8",
+						AffectedPackage: "abc",
+						AffectedVersion: &pkgpb.Version{
 								Name: "0.2.0",
 								Kind: pkgpb.Version_NORMAL,
-							},
-						},
+					        },
+                                                FixedCpeUri:  "cpe:/o:debian:debian_linux:8",
```

Comment:
Looks like this needs to be Go formatted?

Reply:
Done